rollingpin/frontends.py

```diff
@@ -157,20 +165,25 @@ def __init__(self):
         self.old_termio_settings = None
 
     def connectionMade(self):
-        # go to single-character, no-echo mode
+        self.disable_echo()
+
+    def disable_echo(self):
         fileno = sys.stdin.fileno()
         self.old_termio_settings = termios.tcgetattr(fileno)
         tty.setcbreak(fileno)
 
+    def restore_terminal_settings(self):
+        # restore the terminal
```

Comment:
Tiny nit: can probably just drop this comment too, similar to what you did with `disable_echo()`.